### PR TITLE
Add organizationID=ALL query parameter to all the control plane requests

### PR DESCRIPTION
**Pull request by VirajSalaka**
### Purpose
$subject.

### Issues

Fixes #

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [ ] Assigned 'Type' label
- [ ] Assigned the project
- [ ] Validated respective github issues
- [ ] Assigned milestone to the github issue(s)

### CODE REVIEW

**Review by pubudu538 — APPROVED** [review]

File: adapter/internal/common/constants.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org).
```
Comment (renuka-fernando) [inline adapter/internal/common/constants.go:2]:
```suggestion
 *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org).
```

**Review by renuka-fernando — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE1 cluster : dev-deployment-v2 : 20220506.13

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : dev-deployment-v2 : 20220506.13

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE2 cluster : dev-deployment-v2 : 20220506.13

**Comment by choreo-cicd** [comment]
[failed] DataplaneE1 cluster : stage-deployment-v2 : 20220518.3

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE1 cluster : stage-deployment-v2 : 20220518.5

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : prod-deployment-v2 : 20220523.1

**Pull request merged**